docs/AutocompleteInput.md

```diff
@@ -411,7 +411,7 @@ import { PostCreate } from './PostCreate';
 export const App = () => (
     <Admin dataProvider={dataProvider}>
         <Resource name="posts" list={ListGuesser} create={PostCreate} />
-        <Resource name="authors" recordRepresentation="name" />
+        <Resource name="authors" recordRepresentation="last_name" />
```

Comment:
I used `last_name` in this example because `name` is the default, so it makes no sense to set it.